src/cityreader/cityreader.py

```diff
@@ -60,12 +71,43 @@ def cityreader(cities=[]):
 
 # TODO Get latitude and longitude values from the user
 
-def cityreader_stretch(lat1, lon1, lat2, lon2, cities=[]):
+def cityreader_stretch(lat1, lon1, lat2, lon2, cities=cities):
   # within will hold the cities that fall within the specified region
   within = []
 
   # TODO Ensure that the lat and lon valuse are all floats
   # Go through each city and check to see if it falls within 
   # the specified coordinates.
 
+  #given
+  upperright = [lat1, lon1]
+  lowerleft = [lat2, lon2]
+  #calculated
+  upperleft = [lat2, lon1]
```

Comment:
actually brilliant. this makes it so you don't have to flip the coordinates. Nicely done here.